WordPress/src/main/java/org/wordpress/android/ui/domains/management/purchasedomain/PurchaseDomainViewModel.kt

```diff
@@ -97,6 +112,8 @@ class PurchaseDomainViewModel @AssistedInject constructor(
     }
 
     companion object {
+        const val loadingStateAnimationResetDelay = 1000L
```

Comment:
Does it exist to eliminate progress blinking for quick requests?

Reply:
Without the delay the initial state is "flashing" before the checkout webview appears. I tried implementing this with a callback but it got too complicated quickly. I'm open for suggestions on improving this 🙇

Reply:
I'm totally good with this solution! Thanks for the clarification :)